saleor/order/models.py

```diff
@@ -368,6 +368,9 @@ class OrderNote(models.Model):
     content = models.CharField(max_length=250)
     is_public = models.BooleanField(default=True)
 
+    class Meta:
+        ordering = ('-date',)
```

Comment:
Is there an index on this column? If not, sorting can be an expensive operation. Also is date order going to be different than the primary key order?

Reply:
Gonna add index. Right now notes are being displayed as eldest on top and new ones on the bottom. @elwoodxblues mentioned we want to reverse it.

Reply:
If we reverse it, it shouldn't differ from the default ordering by PK and this won't be necessary.

Reply:
As discussed offline, we do not need to set different ordering.